media-libs/shaderc/shaderc-2017.2.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+COMMIT_ID="7a23a01742b88329fb2260eda007172135ba25d4"
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit cmake-multilib python-any-r1 vcs-snapshot
+
+DESCRIPTION="Collection of tools, libraries and tests for shader compilation"
+HOMEPAGE="https://github.com/google/shaderc"
+SRC_URI="https://github.com/google/shaderc/archive/${COMMIT_ID}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples test"
+
+RDEPEND="
+	dev-util/glslang[${MULTILIB_USEDEP}]
+	dev-util/spirv-tools[${MULTILIB_USEDEP}]
+"
+DEPEND="${RDEPEND}
+	${PYTHON_DEPS}
+	dev-util/spirv-headers
+	doc? ( dev-ruby/asciidoctor )
+	test? (
+		dev-cpp/gtest
+		dev-python/nose
```

Comment:
Looks like you really want `python_gen_any_dep` here, and appropriate `python_check_deps()`. Also, do you need python if tests are disabled?

Reply:
>Also, do you need python if tests are disabled?

Yes.